samples/tfm/tfm_cert/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: TF-M certifiable samel
+  name: TF-M Cert
+common:
+    tags: tfm
+    platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf9160dk_nrf9160_ns
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp_ns
+      - nrf9160dk_nrf9160_ns
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
```

Comment:
Here we should a some lines of the output, like:
`Requesting initial attestation token with 64 byte challenge`
`Received initial attestation token of 360 bytes.`

Reply:
Created follow-up task for automated testing of this sample.